### ISSUE TITLE
check-clojure-tools does not work 

### ISSUE THREAD

**Issue by introom** [liquidz/antq#159]
`check-clojure-tools` does not work tools where the installation path is ~/.config/clojure/tools/.   ~/.config is $XDG_CONFIG_HOME

The full command that I found this issue is:
`clojure -Tantq outdated :check-clojure-tools true :upgrade true`

**Comment by liquidz** [comment]
@introom Thanks for your reporting!
I fixed the problem in [bugfix/tool-dir](https://github.com/liquidz/antq/tree/bugfix/tool-dir) branch.
Could you try?

```sh
# install bugfix/tool-dir branch
clojure -Ttools install com.github.liquidz/antq '{:git/url "https://github.com/liquidz/antq" :git/sha "c23871f018c20eff7655f78f2160abdb7cbea510"}' :as antq

# rerun antq as a tool
clojure -Tantq outdated :check-clojure-tools true :upgrade true
```

**Comment by introom** [comment]
I've this in my deps.edn config.
```
           :antq {:extra-deps {com.github.liquidz/antq {:git/url "https://github.com/liquidz/antq"
                                                        :sha "c23871f018c20eff7655f78f2160abdb7cbea510"}}
                  :main-opts ["-m" "antq.core"]}
```

I run the command under some directory that contains a deps.edn file (otherwise antq will complain no project found).
```
 clojure -M:antq :check-clojure-tools true :upgrade true
```

The output is:
```
All dependencies are up-to-date.
```


This file ~/.config/clojure/tools/new.edn is unchanged .

<img width="750" alt="image" src="https://user-images.githubusercontent.com/669045/168187221-94a1f31d-d6d1-457b-9a8d-e4d91d2366d4.png">

**Comment by liquidz** [comment]
@introom You should run `clojure -M:antq --check-clojure-tools --upgrade` when you use `-M`

**Comment by introom** [comment]
Great. It works now.

**Comment by liquidz** [comment]
Thanks for your confirmation! I'll cut a new release this weekend :)

**Comment by liquidz** [comment]
@introom Just released v1.6.2 #162 
Thank you so much!
